keras_nlp/tokenizers/word_piece_tokenizer_test.py

```diff
@@ -41,6 +39,16 @@ def test_dense_output(self):
         self.assertIsInstance(call_output, tf.Tensor)
         self.assertAllEqual(call_output, [[1, 2, 3, 4, 5, 6, 7, 0, 0, 0]])
 
+    def test_vocabulary_size(self):
+        input_data = ["the quick brown fox."]
+        vocab_data = ["[UNK]", "the", "qu", "##ick", "br", "##own", "fox", "."]
+        vocab_size = 5
+        tokenizer = WordPieceTokenizer(
+            vocabulary=vocab_data, vocabulary_size=vocab_size
+        )
+        call_output = tokenizer(input_data)
+        self.assertEqual(tokenizer.vocabulary_size_(), vocab_size)
```

Comment:
why all the `vocabulary_size_` here. Is this a typo?

Reply:
The method `vocabulary_size()` has the same name with `vocabulary_size`. This gives a TypeError when  `vocabulary_size()` is called.

Reply:
Oh got it. We should preserve the `vocabulary_size()` function name, and just leave `self._vocabulary_size` as the variable name.